src/makeCalculator.js

```diff
@@ -1,10 +1,36 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate: (callback, number) => {
+      callback(number);
+
+      return calculator;
+    },
+    add: (number) => {
+      calculator.result += number;
+    },
+    subtract: (number) => {
+      calculator.result -= number;
+    },
+    multiply: (number) => {
+      calculator.result *= number;
+    },
+    divide: (number) => {
+      if (number < 0 || number > 0) {
+        calculator.result /= number;
+      }
```

Comment:
The `divide` method should handle division by zero. Currently, it only checks if the number is not zero but does not handle the case where `number` is zero. Consider adding a condition to prevent division by zero and handle it appropriately, such as returning an error message or keeping the `result` unchanged.